appservice/src/SiteWrapper.ts

```diff
@@ -181,7 +182,12 @@ export class SiteWrapper {
         outputChannel.appendLine(localize('DeleteSucceeded', 'Successfully deleted "{0}".', this.appName));
     }
 
-    public async deploy(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel, configurationSectionName: string, confirmDeployment: boolean = true): Promise<void> {
+    public async deploy(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel, configurationSectionName: string, confirmDeployment: boolean = true, telemetryProperties?: { [key: string]: string }): Promise<void> {
```

Comment:
nit: You could use the `TelemetryProperties` type defined in the ui package. Are you intentionally using a more generic type here?

Reply:
Done